meinberlin/apps/plans/templates/meinberlin_plans/plan_detail.html

```diff
@@ -97,23 +96,66 @@ <h1 class="item-detail__title u-break-word">
                     </div>
                 </div>
 
-                <h3 class="item-detail-2__smalltitle">{% trans 'Contact' %}</h3>
-                {{ object.contact|linebreaks }}
+                <div style="float: left; width: 50%;">
```

Comment:
Ah yes, you were right, we don't put styles into the html. We have classes in the scss components you can either already use or if we need new styles they have to be added there. Let's do that together tomorrow!